client/src/raw.rs

```diff
@@ -980,6 +983,7 @@ proxier! {
         |r| {
             let labels = AttachMetricLabels::namespace(r.get_ref().namespace.clone());
             r.extensions_mut().insert(labels);
+            r.set_timeout(LONG_POLL_TIMEOUT);
```

Comment:
These changes cause the Python test `tests/worker/test_workflow.py::test_workflow_update_timeout_or_cancel` to fail, because the `rpc_timeout` argument in client calls is ignored. See https://github.com/temporalio/sdk-core/pull/781